src/main/kotlin/ch/hevs/cloudio2/cloud/internalservice/CertificateAndPrivateKey.kt

```diff
@@ -1,3 +1,5 @@
 package ch.hevs.cloudio2.cloud.internalservice
 
-class CertificateAndPrivateKey(val certificate: String, val privateKey: String)
+import java.io.Serializable
```

Comment:
Using Java Serializable interface is not optimal, but it is the default when sending objects via AMQP. We should use JSON serialisation instead.